api/aperture/policy/language/v1/policy.proto

```diff
@@ -517,7 +514,7 @@ message Decider {
 
   // Duration of time to wait before a transition to true state.
   // If the duration is zero, the transition will happen instantaneously.
-  google.protobuf.Duration true_for = 4 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+  google.protobuf.Duration positive_for = 4 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
```

Comment:
Not convinced about name change of these properties. `true_for` and `false_for` better represent these properties since they refer to the decider condition being true or false rather than positive or negative. Please revert to the original terminology